lib/client.js

```diff
@@ -0,0 +1,25 @@
+const { createConnection } = require('net');
+const { createInterface } = require('readline');
+ 
+const rl = createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  prompt: '> '
+});
+
+const client = createConnection(4562, () => {
+  console.log('CONNECTED TO SERVER');
+  rl.prompt();
+  rl.on('line', input => {
+    client.write(input);
+    rl.prompt();
+  });
+});
+
+client.on('data', data => {
+  rl.write(data);
```

Comment:
```suggestion
  rl.write(`${data}\n`);
```
Try this code snippet to add a new line to chatroom messages.